hat-psu/hat-psu/BD9E302EFJ-5V1.sch

```diff
@@ -270,6 +270,8 @@ F 0 "C2" H 3592 4046 50  0000 L CNN
 F 1 "10u" H 3592 3955 50  0000 L CNN
 F 2 "Capacitor_SMD:C_1206_3216Metric" H 3500 4000 50  0001 C CNN
 F 3 "~" H 3500 4000 50  0001 C CNN
+F 4 "" H 3500 4000 50  0001 C CNN "DividerTest"
+F 5 "vdiv(5.1, \"(R1+R2)/R2*0.8\", \"E96\", 500e3, 700e3)" H 3500 4000 50  0001 C CNN "DividerTest_expr"
```

Comment:
I'm a bit unclear on how to use this correctly, is it a map you index like `vdiv(...)[2]` to get the `R2` voltage?

Reply:
The function returns a tuple, it's described in the docs of the `voltage_divider` function. You need to store the result to a variable (like `DividerTest` here, but preferrably stored globally) to avoid running the (potentially very long) computation over and over again. What I have indeed not tested is indexing the `evalexpr::Value(TupleType)`, but I sure hope that `evalexpr` supports indexing its own tuples.